packages/app/src/components/RadioGroup.tsx

```
@@ -0,0 +1,57 @@
+              <a href={opt.infoUrl} target="_blank" className="cursor-pointer">
+                <Tooltip content={opt.info}>
+                  <InfoIcon className="h-4 w-4 text-color/50 cursor-pointer" />
+                </Tooltip>
+              </a>
+            ) : (
+              <Tooltip content={opt.info}>
+                <InfoIcon className="h-4 w-4 text-color/50 cursor-default" />
+              </Tooltip>
+            )
+          )}
+        </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)